File: lang/c/loops.c
// FILE_HEADER_START
// 	   File        : loops.c
// 	   Description : Use of loops in C - Sum of 1 to N elements
// 	   Author      : Lavanya Kumar
// FILE_HEADER_END
#include <stdio.h>

int main(int argc, const char *argv[])
{
	int loopCnt = 10, loopItr;
	int loopAddCount = 0;

	// For loop
	for (loopItr = 1; loopItr <= loopCnt; loopItr++) {
		loopAddCount += loopItr;
	}
	printf("Sum of numbers 1 to %d is %d\n", loopCnt, loopAddCount);
	
	return 0;
}

File: lang/golang/loops.go
// FILE_HEADER_START
// 	   File        : loops.go
// 	   Description : Use of loops in Go - Sum of 1 to N numbers
// 	   Author      : Lavanya Kumar
// FILE_HEADER_END

// 'main' package is a must to define main() routine. This is similar
// to 'C' like main() that gets called when program runs.
package main

import "fmt"

func main() {
	// var definesa variable followed by a name followed by the type
	var loopCntSum int = 0;
	
	// := initialize and defines
	// No braces required for a 'for' loop
	for loopItr := 1; loopItr <= 10; loopItr++ {
		loopCntSum += loopItr;
	}
	
	fmt.Println("Sum of 1 to 10 is", loopCntSum);
}
